src/elements/Embed/Embed.scss

```diff
@@ -6,6 +6,19 @@
     flex-direction: column;
     margin: $block-spacing-vertical 0;
 
+    &--expanded {
+        @include media-layout-expanded;
+    }
+
+    &--contained {
+        margin-left: auto;
+        margin-right: auto;
+    }
```

Comment:
Is this necessary? As far as I understand, embeds were always contained before, and it worked. Right?

Reply:
They had `0px` left and right margin:

https://github.com/prezly/content-renderer-react-js/blob/80dd4bf0aef8a19ce9a90b1ddd8460c0cf8dba03/src/elements/Embed/Embed.scss#L7

Plus, I guess having the `--contained` modifier allows for more customization from outside for people using this, so it's good.

Do you think it makes sense to update the Storybook with the new layouts as well?

Reply:
> Do you think it makes sense to update the Storybook with the new layouts as well?

Absolutely! :+1: